models/double_entry_transactions/int_quickbooks__invoice_double_entry.sql

```diff
@@ -152,6 +241,38 @@ invoice_join as (
         and bundle_income_accounts.source_relation = invoice_lines.source_relation
 
     {% endif %}
+
+    {% if var('using_invoice_tax_line', True) and var('using_tax_rate', True) and var('using_tax_agency', True) %}
+    union all
+
+    select 
+        invoice_tax_lines.invoice_id as transaction_id,
+        invoice_tax_lines.source_relation,
+        invoice_tax_lines.index,
+        invoices.transaction_date,
+        invoice_tax_lines.amount,
+        (invoice_tax_lines.amount * coalesce(invoices.exchange_rate, 1)) as converted_amount,
+        'TaxLineDetail' as invoice_line_transaction_type,
+        tax_account_join.account_id,
+        cast(null as {{ dbt.type_string() }}) as class_id,
```

Comment:
Should we apply the same invoices.class_id here? Similar to how the same is done for customer and department below?

Reply:
Wasn't 100% certain if that was the way it's done (had trouble finding supporting documentation), but  we could try it and see. Implemented.